### ISSUE TITLE
dead?

### ISSUE THREAD

**Issue by ghost** [opengapps/apkcrawler#77]
is this project dead? seems no updates in two years

**Comment by tttrinhtran** [comment]
did you have any problem running it?

**Comment by ghost** [comment]
did you?

**Comment by tttrinhtran** [comment]
> did you?
yes i had problem like issue #70. Is it true that we need to download report_sources.sh in the same directory and run it? then running the command ./report_sources.sh nosig | apkmirrorcrawler.py ? 
btw i am trying to crawl APK from APKMirror
